hack/e2e/run-rp-and-e2e.sh

```diff
@@ -65,9 +65,36 @@ register_sub() {
 }
 
 run_e2e() {
+    if [ "$RP_MODE" != "development" ]; then
+    export TEST_E2E="docker run \
+                --network host \
+                --env-file <(env | egrep 'AZURE|E2E|ARO') \
+                --env LOCATION \
+                --env CI \
+                --env RESOURCEGROUP \
+                --env CLUSTER \
+                --env RP_MODE \
+                --entrypoint e2e.test $ARO_IMAGE"
+    RESOURCEGROUP=$ARO_RESOURCEGROUP TEST_E2E=$TEST_E2E make test-e2e
+    else
     RESOURCEGROUP=$ARO_RESOURCEGROUP make test-e2e
+    fi
```

Comment:
Not sure how better reuse this dam function in other scripts. tempted to rip it apart as It almost non existing now.